README.md

```diff
@@ -80,4 +80,71 @@ pipenv install --dev
 
 # Setup pre-commit and pre-push hooks
 pipenv run pre-commit install -t pre-commit
+## pulling_ace/utils/subprocessor.py
+
+The `pulling_ace/utils/subprocessor.py` script is a Python script that contains several functions related to toxicity and prompt injection. These functions are used to run the Garak tool with different probes and evaluate the robustness of natural language processing models.
```

Comment:
Make it more general . Example : you can toxicity and prompt injection in your Hugginface model using garak probes calling --toxicity or --promptinjection

Reply:
change this

Reply:
Thanks. Redo the task in the README.md file. Explain the functionality of the subprocess.py implementation, not the functions in the script. Read the script cli.py to create an example. I need it to be more functional. Erase the functions description .

Reply:
Change the README.md . Redo the task